lib/bridge.js

```diff
@@ -83,7 +83,7 @@ Bridge.prototype.list = function list () {
       return nukis;
     })
     .map(function processEachNuki (nuki) {
-      nuki.nuki = new Nuki(self, nuki.nukiId, {delayBetweenRequests: self.delayBetweenRequests});
+      nuki.nuki = new Nuki(self, nuki.nukiId, nuki.deviceType || 0);
```

Comment:
So I guess `deviceType` is coming from the list, right?

Reply:
Yes, `deviceType` is retrieved from `/list` as documented in the updated Nuki Bridge API docs, see https://developer.nuki.io/page/nuki-bridge-http-api-190/4#heading--list.
- 0 => smartlock (Nuki Smart Lock)
- 2 => opener (Nuki Opener)